docs/user/tutorials/01-30-30-azure-vpc-peering.md

```diff
@@ -16,12 +16,11 @@ This tutorial explains how to create a Virtual Private Cloud (VPC) peering conne
     az account set --subscription $SUBSCRIPTION
     ```
 
-2. Assign the required roles to the Cloud Manager peering service principal:
-
+2. Assign the required roles to the Cloud Manager peering service principal. See [AzureVpcPeering Custom Resource](../resources/04-30-30-azure-vpc-peering.md#authorization) to identify Cloud Manager service principal: 
```

Comment:
I would create a separate doc on the Authorization, etc. instead of having everything in the resources document. We could use the details from the Authorization section as input. TBD in the next PR.